.drone.star

```diff
@@ -224,10 +224,88 @@ config = {
 			'extraEnvironment': {
 				'S3_TYPE': 'scality',
 			},
+			'scalityS3': True,
 			'federatedServerNeeded': True,
 			'runCoreTests': True,
 			'runAllSuites': True,
-			'numberOfParts': 32,
+			'numberOfParts': 8,
```

Comment:
I reduced the number of pipelines, because actually this is only running the smoke test scenarios, so we do not need to split into so many parts.